runtest.py

```diff
@@ -298,7 +311,7 @@ def main():
     arguments = docopt(__doc__)
     run_test_obj = RunTest(exec_config=arguments['--exec-config'], firefox_config=arguments['--firefox-config'],
                            index_config=arguments['--index-config'], online_config=arguments['--online-config'],
-                           global_config=arguments['--global-config'])
+                           global_config=arguments['--global-config'], chrome_config=arguments['--chrome-config'])
```

Comment:
It seems like we should add the `CHROME_CONFIG` into https://github.com/Mozilla-TWQA/Hasal/blob/dev/lib/baseTest.py#L162-L194